src/js/player.js

```diff
@@ -354,6 +354,9 @@ class Player extends Component {
     // create logger
     this.log = createLogger(this.id_);
 
+    // Hold our own reference to fullscreen api so it can be mocked in tests
+    this.fsApi = FullscreenApi;
```

Comment:
guessing `_` indicates the member is "private"?

The `fsApi` member is accessed from the `FullscreenToggle` class - do you still want this to be suffixed with `_`?

I added a line note where it is accessed

Reply:
Yeah, private, though to the library rather than to the class/object; it's fine if the FullscreenToggle uses it. We just don't want to officially expose it publicly as part of this PR. The suffix isn't perfect, but it's a good enough convention.